training/frameworks/pytorch/lightning/single_node_single_device_mnist.ipynb

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
<p>In general, add more descriptive text and markdown explaining what this notebook accomplishes, what customers will learn, and potential extensions or applications.</p><p> </p><p>Are there endpoints or other resources that need to be deleted to avoid incurring ongoing costs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>